src/frontend/skipComment.sk

```diff
@@ -23,9 +23,11 @@ extension class Comment {
   // The contents of the comment not including leading //, /* or trailing */
   // Does include the trailing NewLine for // comments if present.
   fun contents(): String {
+    v = this.value;
     this.kind match {
-    | LineComment() -> this.value.sub(2, this.value.length() - 2)
-    | DelimitedComment() -> this.value.sub(2, this.value.length() - 4)
+    | LineComment() -> v.getIter().forward(2).takeString()
```

Comment:
This pattern shows up a bunch now: get iterator, advance forward by N chars, then "take" either to the end or to some end iterator (that is itself created by getting the end iterator and reversing M steps). Seems like we should make slicing from a start to negative-end offset easier.